stl/inc/complex

```diff
@@ -186,10 +200,22 @@ public:
         return (numeric_limits<long double>::max)();
     }
 
+    static _Ty _Abs(_Ty _Left) {
+        // this always returns the correct value

```

Comment:
"this always returns the correct value" is not a useful comment - it implies that every function in the library without such a comment doesn't always return the correct value. (Several occurrences)